packages/salesforcedx-vscode-soql/package.json

```
@@ -43,7 +43,7 @@
     "jsforce": "^1.10.0",
     "papaparse": "^5.3.0",
     "vscode-extension-telemetry": "^0.1.6",
-    "vscode-languageclient": "^6.1.3"
+    "vscode-languageclient": "^5.2.1"
```

```suggestion
    "vscode-languageclient": "6.1.3",
    "vscode-languageserver-protocol": "3.15.3"
```

With that change it works and we keep the same version we are using.